pages/basics/installation.md

```diff
@@ -106,18 +105,18 @@ You can change it on the `julia env` toolbar option.
 ## Pkg - Install packages
 
 Julia comes with a built-in package manager: `Pkg`.
-`Pkg` is itself a package, but it's part of the stdlib, so it's always available.
-It is a very robust manager, that deals with conflicts and can handle environments.
-`Pkg` connects to the official Julia registry to resolve package names.
-However, it also allows for unregistered packages to be installed.
+`Pkg` is itself a package and 
+- it's always available because it's part of the stdlib;
+- it's also a very robust manager, that deals with conflicts and can handle environments.
+- connects to the official Julia registry to resolve package names.
+- :( it also allows for unregistered packages to be installed.
```

Comment:
```suggestion
- `Pkg` is itself a package;
- it's always available because it's part of the stdlib;
- it's a very robust manager, that deals with conflicts and can handle environments;
- it connects to the official Julia registry to resolve package names;
- it also allows for unregistered packages to be installed.
```